epi/src/lib.rs

```diff
@@ -319,6 +319,11 @@ impl Frame {
         self.lock().output.drag_window = true;
     }
 
+    /// If you need to request a repaint from another thread, send it to that other thread.
+    pub fn repaint_signal(&self) -> Arc<dyn backend::RepaintSignal> {
```

Comment:
a bad idea to expose _backend_ stuff here?
should it be done in some other way?